README.md

```diff
@@ -50,27 +46,27 @@ Beat Saber open replay format.
 
 1                              - byte, frames array start.
 framesCount                    - int, frames count.
-{                              - Frame structure
-  time                         - float, song time
-  fps                          - int, player's FPS
-  {                            - Head structure
+{                              - Frame structure.
+  time                         - float, song time.
+  fps                          - int, player's FPS.
+  {                            - Head structure.
     {x, y, z}                  - 3 floats, position.
-    {x, y, z, w}               - 4 floats, rotation.
+    {x, y, z, w}               - 4 floats, rotation (in quaternion form).
```

Comment:
idk if it's necessary to specify when rotation has 4 floats and there are few more quaternions further the file